src/lib/utils/array.ts

```diff
@@ -0,0 +1,3 @@
+export function allEqual<T>(array: T[]): boolean {
```

Comment:
Could you not just have used a lodash function for this?

Reply:
hm, seems there is no one for this?

Reply:
I actually didn't look at the internal logic. This naming doesn't make sense. The logic is 'Is every element equal to the first element, right? I wouldn't extract this into its own function, just use the `every` in place.